client/blocks/login/login-form.jsx

```diff
@@ -259,11 +265,14 @@ export class LoginForm extends Component {
 					</div>
 
 					{ isOauthLogin && (
-						<div className="login__form-signup-link">
+						<div className={ classNames( {
+							'login__form-signup-link': true,
+							disabled: ! this.props.oauth2ClientData.signupUrl
+						} ) }>
 							{ this.props.translate( 'Not on WordPress.com? {{signupLink}}Create an Account{{/signupLink}}.',
 								{
 									components: {
-										signupLink: <a href={ config( 'signup_url' ) } />,
+										signupLink: <a href={ this.props.oauth2ClientData.signupUrl } />,
```

Comment:
Ridiculously minor but since we use ` this.props.oauth2ClientData` in a few places in this method we could use destructuring to save us from duplication.